components/IncidentForm.jsx

```diff
@@ -31,16 +31,23 @@ export default function IncidentForm({setData}) {
     const response = await fetch(endPoint, options)
 
     const result = await response.json()
-    setData(result)
+    //{incidentReport_id: 24, incidentReport_note: 'sdfsd'}
+    
+    setData([
+      ...data, 
+      {
```

Comment:
This doesn't handle the case where another person submits an incident report between the time that you fetched incident reports, and the time where you post a new one.

Might be better to pass a function into this component that fetches all the incident reports.

so something like 

const response = await fetch(endPoint, options);
const result = await response.json();
await getIncidentReports();